pkg/test/helm/helm.go

```diff
@@ -39,19 +41,30 @@ func Init(homeDir string, clientOnly bool) error {
 }
 
 func Template(homeDir, template, name, namespace string, valuesFile string, values map[string]string) (string, error) {
-	// Apply the overrides for the values file.
-	valuesString := ""
-	for k, v := range values {
-		valuesString += fmt.Sprintf(" --set %s=%s", k, v)
-	}
-
 	valuesFileString := ""
 	if valuesFile != "" {
 		valuesFileString = fmt.Sprintf("--values %s", valuesFile)
 	}
 
-	out, err := shell.Execute("helm --home %s template %s --name %s --namespace %s %s %s",
-		homeDir, template, name, namespace, valuesFileString, valuesString)
+	str := fmt.Sprintf("helm --home %s template %s --name %s --namespace %s %s",
```

Comment:
If you just make this an array to begin with it will he much simpler than creating a string and splitting it

Reply:
Done.